modules/renter/downloadcache.go

```diff
@@ -0,0 +1,53 @@
+package renter
+
+import "time"
+
+// addChunkToCache adds the chunk to the cache if the download is a streaming
+// endpoint download.
+func (udc *unfinishedDownloadChunk) addChunkToCache(data []byte) {
```

Comment:
Should add a TODO for this function that deletes the least-recently-accessed part of the cache instead of just deleting whatever map element is created first. No need to do that for this PR.

@tbenz9 this is probably a pretty beginner-friendly issue that we can let some newer contributor tackle.